addons/project_sms/security/project_sms_security.xml

```diff
@@ -4,7 +4,7 @@
         <field name="name">SMS Template: project manager CUD on project/task</field>
         <field name="model_id" ref="sms.model_sms_template"/>
         <field name="groups" eval="[(4, ref('project.group_project_manager'))]"/>
-        <field name="domain_force">[('model_id.model', 'in', ('project.task.type', 'project.project.stage'))]</field>
+        <field name="domain_force">[('model_id.model', 'in', ('project.task', 'project.project'))]</field>
```

Comment:
```suggestion
        <field name="domain_force">[('model', 'in', ('project.task', 'project.project'))]</field>
```

we could directly use `model` field defined in `sms.template`, it is stored, so it will increase the perf at the same time. 😁